MIP113/MIP113.md

```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
-| Jiaozi | [Address](https://etherscan.io/address/0x774FB7955Ff69E5b8A5AFD0C0833B2719f7a2636)<br>[Signature](https://etherscan.io/verifySig/242924) | [Contract](https://etherscan.io/address/0xc1a1bdb7d60b7cb3920787a15a2b583786c52fb6)<br>[Signature](https://etherscan.io/verifySig/243693) | No active Contract 2 | [Link](https://forum.makerdao.com/t/jiaozi-ad-recognition-submission/24374) |
+| Delegate Name | EA Address | Delegation Contract | Forum Post |
+|---|---|---|---|
+| vigilant | [Address](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61)<br>[Signature](https://etherscan.io/verifySig/16682) | [Contract](https://etherscan.io/address/0xDE08aEf2b221274231b3547491ec8f0Fc80917e1)<br>[Signature](https://etherscan.io/verifySig/255532) | [Link](https://forum.makerdao.com/t/cd-recognition-submission-vigilant/20457) |
+| QGov | [Address](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6)<br>[Signature](https://etherscan.io/verifySig/16763) | [Contract](https://etherscan.io/address/0x0a907fe3adb890db7db27a7f21e188a4127b2e7c)<br>[Signature](https://etherscan.io/verifySig/16765) | [Link](https://forum.makerdao.com/t/qgov-cd-recognition-submission/20494) |
+| Bonapublica | [Address](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3)<br>[Signature](https://etherscan.io/verifySig/21271) | [Contract](https://etherscan.io/address/0xe2bfDa5e1f59325E4b8dF5feaA30e4aB6516bf28)<br>[Signature](https://etherscan.io/verifySig/16766) | [Link](https://forum.makerdao.com/t/bonapublica-constitutional-delegate-recognition-submission/20451) |
+| PBG | [Address](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2)<br>[Signature](https://etherscan.io/verifySig/16773) | [Contract](https://etherscan.io/address/0x69b576a7e193a15a570ee5bb2149deb3f03537a2)<br>[Signature](https://etherscan.io/verifySig/21570) | [Link](https://forum.makerdao.com/t/pbg-constitutional-delegate-communication-platform/20471) |
+| Pf | [Address](https://etherscan.io/address/0x94A388BEC0769c3394A48dB4DaBE5F29994100B3)<br>[Signature](https://etherscan.io/verifySig/16824) | [Contract](https://etherscan.io/address/0x62060879dfBB6dEf3B73CFC48f1f0595c0FeD505)<br>[Signature](https://etherscan.io/verifySig/16825) | [Link](https://forum.makerdao.com/t/pf-recognition-submission/20520) |
+| UPMaker | [Address](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd)<br>[Signature](https://etherscan.io/verifySig/16997) | [Contract](https://etherscan.io/address/0xc241a035bc98128ac3983812a67dfc52ddeae09a)<br>[Signature](https://etherscan.io/verifySig/38946) | [Link](https://forum.makerdao.com/t/upmaker-cd-recognition-submission/20577) |
```

Comment:
Delegate asked to keep other contract (the one with MKR delegated to it). source: https://forum.makerdao.com/t/upmaker-ad-submission/20577/80

source for signature: https://forum.makerdao.com/t/upmaker-ad-submission/20577/59
```suggestion
| UPMaker | [Address](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd)<br>[Signature](https://etherscan.io/verifySig/16997) | [Contract](https://etherscan.io/address/0xc9C17EE58810172622b43B09C9C9FF31e50f5179)<br>[Signature](https://etherscan.io/verifySig/39034) | [Link](https://forum.makerdao.com/t/upmaker-cd-recognition-submission/20577) |
```